src/main/java/gregtech/api/metatileentity/SimpleMachineMetaTileEntity.java

```diff
@@ -455,13 +470,15 @@ private void circuitConfigPlus(Widget.ClickData data) {
         ItemStack stack;
         if (circuitInventory != null && IntCircuitIngredient.isIntegratedCircuit(stack = circuitInventory.getStackInSlot(0))) {
             IntCircuitIngredient.adjustConfiguration(stack, data.isShiftClick ? 5 : 1);
+            this.workable.forceRecipeRecheck();
```

Comment:
Instead of forcing a recipe recheck, it might be better to do

```java
this.notifiedItemInputList.add(circuitInventory);
this.notifiedFluidInputList.add(getImportFluids());
```

Reply:
Does the notified list get reset?
Why does the fluid list need to be updated?

Reply:
Yep, it is cleared
![image](https://user-images.githubusercontent.com/18713839/186520439-e055e237-16a9-43c0-8d90-f2d2ada05ee3.png)

Reply:
There are some machines that add special logic earlier on in the notifiable chain, specifically `shouldSearchForRecipes`. None of these machines have a circuit slot currently, but that might not be the case in the future, so I want to ensure that any custom notifiable logic that pops up is accounted for by going through the notifiable chain, instead of directly forcing a recipe recheck.

As for the fluids, that was me thinking that we also had to notify the fluid inventories to get them included in a recipe lookup, but that is not actually the case for singleblock machines, so the fluid notification is probably not needed.